usps-processor@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=USPS Processor
+
+After=docker.service
+Requires=docker.service
+After=consul@%i.service
+Wants=consul@%i.service
+After=wildfly@%i.service
+Requires=wildfly@%i.service
```

Comment:
Since this requires RabbitMQ too, can you add `After=rabbitmq@%i.service` and `Requires=rabbitmq@%i.service`?